pkg/virtctl/expose/expose_test.go

```diff
@@ -196,9 +159,18 @@ var _ = Describe("Expose", func() {
 			}
 		}
 
+		verifySelector := func(resType string, selector map[string]string) {
```

Comment:
Any chance we can avoid asserting in a helper function? Perhaps returning something that can be asserted at the test level?

But if you must, I would also prefer a regular function and not a lambda one.

Reply:
ACK, updated to address this.